### PR TITLE
Add prod index support for multiple widgets

### PR DESCRIPTION
**Pull request by eetay**

### CODE REVIEW

File: bin/widget-commands.js [hunk 1]
```diff
@@ -41,10 +49,19 @@ if (!parsedArgs.widget) {
...
-function create_widget(widget) {
+const widgetDevUrl = widget => `${webpack_config.devServer.https ? 'https' : 'http'}://localhost:${webpack_config.devServer.port}`
+const widgetProdUrl = widget => `/platform_widgets/${widget}/index.html`
+
+function createOrUpdateWidget(widget, prod) {
   const create_widget = SamanageAPI.create('platform_widget', 'admin')
+  const widgetConfig = config.widgets[parsedArgs.widget]
   const create_widget_request = create_widget(
-    Object.assign({}, config.widgets[parsedArgs.widget], { code: `${webpack_config.devServer.https ? 'https' : 'http'}://localhost:${webpack_config.devServer.port}` })
+    Object.assign({},
```
Comment (Yuli-Arbov) [inline bin/widget-commands.js:59]: redundant empty object

File: bin/widget-commands.js [hunk 2]
```diff
@@ -57,6 +74,26 @@ function create_widget(widget) {
...
 }
 
-if (parsedArgs.command == 'create-widget') {
-  create_widget(parsedArgs.widget)
+if (parsedArgs.command == 'create-widget' || parsedArgs.command == 'update-widget') {
+  createOrUpdateWidget(parsedArgs.widget)
+}
+else if (parsedArgs.command == 'deploy-widget') {
+  const targetDir = '../Frontend/public/platform_widgets'
+  var targetDirExists = false
```
Comment (Yuli-Arbov) [inline bin/widget-commands.js]: please change "var" to "let"

File: webpack/webpack.prod.config.js [hunk 3]
```diff
@@ -1,15 +1,29 @@
-var webpack = require('webpack')
-var path = require('path')
+//const UglifyJsPlugin = require('uglifyjs-webpack-plugin')
```
Comment (Yuli-Arbov) [inline webpack/webpack.prod.config.js]: do we need the comments?

File: bin/widget-commands.js (hunk 1 above)
Comment (eetay) [reply bin/widget-commands.js:59]: not redundant as explained

**Review by Yuli-Arbov — APPROVED** [review]

**Comment by eetay** [comment]
against wrong branch

**Pull request closed without merging**